iconservice/iconscore/icon_score_loader.py

```diff
@@ -52,17 +53,17 @@ def make_score_path(self, score_addr: 'Address', tx_hash: 'bytes') -> str:
         return path.join(self._score_root_path, score_addr.to_bytes().hex(), converted_tx_hash)
 
     def try_score_package_validate(self, whitelist_table: dict, score_path: str):
-        pkg_root_import: str = self._make_pkg_root_import(score_path)
-        ScorePackageValidator().execute(whitelist_table, score_path, pkg_root_import)
+        pkg_import_root: str = self._make_pkg_root_import(score_path)
```

Comment:
If you modified the var name to `pkg_import_root`, the method name should have also been modified to `_make_pkg_import_root()` accordingly.

Reply:
fix name "pkg_root_package"